index.html

```diff
@@ -3,12 +3,36 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
+    <title>할 일 목록 프로그램</title>
     <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/c9ad708a77.js" crossorigin="anonymous"></script>
   </head>
 
   <body>
-    <div class="container"></div>
+    <main>
+    <div class="container">
+      <div class="todoInput">
+        <div class="todoTitle">
```

Comment:
HTML 태그의 id/class 네임은 보통 `kebab-case`로 짓습니다!

```suggestion
    <div class="container">
      <div class="todo-input">
        <div class="todo-title">
```